pulp_container/app/registry_api.py

```diff
@@ -309,6 +312,12 @@ def get_pull_through_drv(self, path):
         if not pull_through_cache_distribution:
             raise RepositoryNotFound(name=path)
 
+        pull_through_remote = models.ContainerPullThroughRemote.objects.get(
+            pk=pull_through_cache_distribution.remote
+        )
+        if not filter_resource(path, pull_through_remote.includes, pull_through_remote.excludes):
+            raise RepositoryNotFound(name=path)
```

Comment:
I think we can access the pull-through remote "directly". Then, we can also pass `pull_through_cache_distribution.remote` to `_get_pull_through_remote_data`.

Reply:
`pull_through_cache_distribution.remote` is of type `Remote` (not `ContainerPullThroughRemote`) ... so there is no includes/excludes definition in it :cry:

Reply:
You can call cast() on it to get the correct type.